lib/did_you_mean/extra_features.rb

```diff
@@ -14,3 +14,4 @@ module DidYouMean
 
 require 'did_you_mean/extra_features/initializer_name_correction'
 require 'did_you_mean/extra_features/ivar_name_correction'
+require 'did_you_mean/extra_features/key_error'
```

Comment:
Let's rename the file name `key_error.rb` to `key_name_correction.rb` since it would probably read better.